src/ui/core.c

```diff
@@ -1079,51 +1078,35 @@ ui_contact_offline(char* barejid, char* resource, char* status)
 void
 ui_clear_win_title(void)
 {
-    printf("%c]0;%c", '\033', '\007');
+    fputs("\e]0;\a", stdout);
```

Comment:
What's the advantage of this?

Reply:
It's slightly more efficient. ie, there's no reason here to use a printf to construct a string when the args are constants.

Reply:
Ah of course. Sorry I was confused due to different languages, thinking about print and printf.